docs/Rootless.md

```diff
@@ -79,4 +91,4 @@ else
 	ARCHS = armv7 armv7s arm64 arm64e
 	TARGET = iphone:clang:latest:7.0
 endif
-```
+```
```

Comment:
Unrelated, please revert

Reply:
Can you please clarify a bit more on this?

Reply:
Yep; I meant that the change here is unrelated to the rest of the PR, so it should not be included.